packages/api/src/utils/stripe.js

```diff
@@ -429,7 +437,8 @@ export function createStripeBillingContext (env) {
   return {
     billing,
     customers,
-    subscriptions
+    subscriptions,
+    termsOfService
```

Comment:
Am I correct in understanding that the TermsOfServiceService doesn't depend at all on stripe?

If so, I think it's best that we don't include it in the return of this function. I think the way our tests get set up almost always have access to a sql db, so we don't even need to use a mock a lot of the times e.g. when integration testing the http api layer.

The main reason I used so many mocks for the stripe-specific stuff is because I didn't want our tests to have to invoke the stripe.com APIs (so I used mocks), whereas I think it's frequently okay to rely for a lot of our tests to rely on the sql db.

We can rename some things if it stop being intuitive, but the way I'd think about it is maybe the TermsOfService stuff isn't necessarily about 'BillingContext' as much as a separate kind of context that's a bit orthogonal (but often used together). The distinction being that everything in the BillingContext can be fulfilled by stripe.com, whereas the AgreementsContext doesn't use stripe at all and just uses a sqldb. (if in practice AgreementsContext only has one service in it, we might not need a 'context' interface at all)

Reply:
It still feels odd to me to construct the agreements service in this stripe-specific context, but I spent some time refactoring it and hit my timebox. This'll work well in practice, and we can refactor if/when needed.